### PR TITLE
OSX compatibility with grpc 0.13 integration

### PR DESCRIPTION
**Pull request by Rastusik**
[Review on Reviewable](https://reviewable.io/reviews/youtube/vitess/1538)

### CODE REVIEW

**Comment by michael-berlin** [comment]
Thanks for the fixes :) Let's hold off on it shortly and and get my gRPC Python installation fixes in first: https://github.com/youtube/vitess/pull/1541

It's not done yet as some platforms are still broken. Once it's done, I'll let you know. After that, can you please adjust your PR?

File: bootstrap.sh [hunk 1]
```diff
@@ -89,6 +89,7 @@ else
 
   if [ `uname -s` == "Darwin" ]; then
     # on OSX tox is installed in the following path
+    export PYTHONPATH=$(prepend_path $PYTHONPATH $grpc_dist/usr/local/lib/python2.7/site-packages)
```
Comment (michael-berlin) [inline bootstrap.sh]: After https://github.com/youtube/vitess/pull/1541 "tox" should no longer be required and this path shouldn't be on $PYTHONPATH

Can you please check the content of the site-packages directory on your system? It should be empty when you do a fresh bootstrap (delete $grpc_dist completely).

**Comment by Rastusik** [comment]
@michael-berlin sure, no problem, I pushed the adjustment

**Comment by michael-berlin** [comment]
https://github.com/youtube/vitess/pull/1541 is in now.

Can you please rebase your change on that one?

After that, I'm happy to merge your change.

**Comment by Rastusik** [comment]
Did you mean like this? ^^

**Comment by michael-berlin** [comment]
No. It looks like you cherry-picked my temporary commits now.

Instead, can you just rebase your commit on the latest master?

I'm not sure how much familiar you're with git, but I would probably do it this way:

```sh
git rebase -i HEAD~10
```

This will represent you the list of the 10 last commits. Delete all "WIP: ..." ones and leave only yours.

After that, rebase your change on top of the current master:

```git rebase master```

(assuming that "master" points to a recent local branch/copy of the master branch from github.com/youtube/vitess)

**Comment by Rastusik** [comment]
Well, I know how to rebase, I just didn't understand on top of what I was supposed to rebase :) Is this ok now?

**Comment by michael-berlin** [comment]
Sorry for not being clear enough :)

Did you test it on your OSX machine? Does this work? Once you confirmed, I'll merge it.

**Comment by Rastusik** [comment]
Yes, I just rebuilt all the dependencies, works fine.

**Comment by michael-berlin** [comment]
LGTM

**Pull request merged**